dbt/task/compiler/linker.py

```diff
@@ -0,0 +1,93 @@
+import sqlparse
```

Comment:
@cmerrick I imagine we're going to want to do much more in the compilation step than we're currently doing. For instance, sort/dist keys, column encodings, etc can all be interpolated at compile time. I think it makes sense to leave the actual sql parsing code in a separate dir for now. I did change the name from `compile` to `build`